pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -37,23 +39,81 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
       {!getComponents.isLoading && component && (
         <>
           <div className={styles.heading}>
-            <div>
+            <div className={styles.context}>
               <Heading1>{component.name}</Heading1>
-              <span className={styles.subtitle}>Phasellus tempus. Aenean vulputate eleifend tellus. Sed a libero.</span>
+              <span className={styles.subtitle}>Component organisatie</span>
+              <Paragraph className={styles.description}>{component.description}</Paragraph>
+              <div className={styles.tags}>
+                <Tag tag={component.intendedAudience} />
+                <Tag tag={component.intendedAudience} />
+              </div>
             </div>
-            <div className={styles.tags}>
-              <Tag tag={component.intendedAudience} />
-              <Tag tag={component.developmentStatus} />
+
+            <div className={styles.object}>
+              <img src={grey} className={styles.img} />
+              <Button icon={<ExternalLinkIcon />}>Toevoegen aan catalogus</Button>
             </div>
           </div>
 
-          <Paragraph className={styles.description}>{component.description}</Paragraph>
+          <div className={styles.cards}>
+            <InfoCard title={component.name} content="Info for install component" />
+            <InfoCard title="Github" content="Repo en links" />
+            <InfoCard title="Domedata" content="Informasi for domedata " />
+            <InfoCard title="Domedata" content="Informasi for domedata " />
+          </div>
+
+          <Table>
+            <TableBody>
+              <TableRow>
+                <TableHeader>Gemma</TableHeader>
+                <TableCell>Info gemma</TableCell>
+              </TableRow>
+              <TableRow>
+                <TableHeader>UPL</TableHeader>
+                <TableCell>Info UPL</TableCell>
+              </TableRow>
+              <TableRow>
+                <TableHeader>Standaarden</TableHeader>
+                <TableCell>Info Standaarden</TableCell>
+              </TableRow>
+              <TableRow>
+                <TableHeader>Wet en regelgeving</TableHeader>
+                <TableCell>Info Wet en regelgeving</TableCell>
+              </TableRow>
+            </TableBody>
+          </Table>
 
-          <a className={styles.externalLink} href={component.isBasedOn} target="_blank">
-            <Link icon={<ArrowRightIcon />} iconAlign="start">
-              {t("View component on GitHub")}
-            </Link>
-          </a>
+          <div>
+            <TabContext value={component.toString()}>
```

Comment:
Because these tabs do not fit the screen, and we still have time, I think it's best to try and make this vertical instead of horizontal.

Reply:
Tab component is alleen horizontaal kunnen gebruiken. als je wil verticaal, wij moet nieuw component maken.
https://nl-design-system.github.io/denhaag/?path=/docs/react-navigation-tab--default